staging/src/k8s.io/client-go/transport/config.go

```diff
@@ -131,10 +131,11 @@ func (c *Config) Wrap(fn WrapperFunc) {
 
 // TLSConfig holds the information needed to set up a TLS transport.
 type TLSConfig struct {
-	CAFile         string // Path of the PEM-encoded server trusted root certificates.
-	CertFile       string // Path of the PEM-encoded client certificate.
-	KeyFile        string // Path of the PEM-encoded client key.
-	ReloadTLSFiles bool   // Set to indicate that the original config provided files, and that they should be reloaded
+	CAFile              string // Path of the PEM-encoded server trusted root certificates.
+	CertFile            string // Path of the PEM-encoded client certificate.
+	KeyFile             string // Path of the PEM-encoded client key.
+	ReloadTLSFiles      bool   // Set to indicate that the original config provided files, and that they should be reloaded
+	CallerHandlesDialer bool   // TODO
```

Comment:
I'm assuming this is still WIP with these TODOs in place

Reply:
I am unsure if `CallerHandlesDialer` actually adds value...